### PR TITLE
Added automatic tracking sessions configurations

### PR DESCRIPTION
**Pull request by SeqLaz**
Added documentation for enabling automatically start Tracking session on load project

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3883 #5062 Project tracking sessions support](https://app.clickup.com/t/2192114/QF-3883)

File: documentation/how-to/tracking.en.md [hunk 1]
```diff
@@ -69,3 +69,22 @@ on a legend layer linked to an active tracking session and select the
...
+
+## Project Tracking Sessions Auto-Start Feature
+
+The feature enables tracking sessions to automatically start upon project load, with the option for users to fill out a feature form unless the 'Hide Form on Add Feature' toggle is enabled for the project's layer, but the default attribute values are used.
+
+!![Hide Form on Added Feature](../assets/images/hide-form-on-add-feature.png)
+
+### Implementation
+
+For implement it enter to the layer properties and in the QField panel activate "Tracking Session" and specify the your requirement for the tracking.
```
Comment (nirvn) [inline documentation/how-to/tracking.en.md:81]:
```suggestion
The configuration of a project tracking session happens in the vector layer properties dialog's QField panel. There, you can activate "Tracking Session" and specify the tracking requirements. Note that the QField panel is only available when the qfieldsync plugin is installed in your QGIS profile.
```

File: documentation/how-to/tracking.en.md [hunk 2]
```diff
@@ -69,3 +69,22 @@ on a legend layer linked to an active tracking session and select the
...
 !![](../assets/images/tracking-stop.png)
+
+## Project Tracking Sessions Auto-Start Feature
+
+The feature enables tracking sessions to automatically start upon project load, with the option for users to fill out a feature form unless the 'Hide Form on Add Feature' toggle is enabled for the project's layer, but the default attribute values are used.
+
+!![Hide Form on Added Feature](../assets/images/hide-form-on-add-feature.png)
+
+### Implementation
+
```
Comment (nirvn) [inline documentation/how-to/tracking.en.md:80]:
```suggestion
:material-monitor: Desktop preparation

```

**Comment by nirvn** [comment]
@SeqLaz , thanks for the speedy documentation of what I feel is a pretty darn good QField feature! Glad our users will have a higher  chance of stumbling on this functionality by reading the manual.

File: documentation/how-to/tracking.en.md [hunk 3]
```diff
@@ -69,3 +69,22 @@ on a legend layer linked to an active tracking session and select the
 *Stop tracking* button.
 
 !![](../assets/images/tracking-stop.png)
+
+## Project Tracking Sessions Auto-Start Feature
```
[the remark this answers is not in the document]
Comment (SeqLaz) [reply documentation/how-to/tracking.en.md:73]: Much better!

File: documentation/how-to/tracking.en.md [hunk 4]
```diff
@@ -69,3 +69,22 @@ on a legend layer linked to an active tracking session and select the
...
+
+The feature enables tracking sessions to automatically start upon project load, with the option for users to fill out a feature form unless the 'Hide Form on Add Feature' toggle is enabled for the project's layer, but the default attribute values are used.
```
[the remark this answers is not in the document]
Comment (SeqLaz) [reply documentation/how-to/tracking.en.md:75]: Great!

File: documentation/how-to/tracking.en.md [hunk 5]
```diff
@@ -69,3 +69,23 @@ on a legend layer linked to an active tracking session and select the
...
+## Predefined Project Tracking Sessions
+
+This functionality enables one or more position tracking sessions to automatically start upon project load. Users will be presented with a feature form popup as the tracking session begins to fill in attributes. If the vector layer used to track has the 'Hide Form on Add Feature' mode selected, the feature form will be skipped. 
+
+!![Hide Form on Added Feature](../assets/images/hide-form-on-add-feature.png)
+
+### Configure a Project Tracking Session
+:material-monitor: Desktop preparation
+
+The configuration of a project tracking session happens in the vector layer properties dialog's QField panel. There, you can activate "Tracking Session" and specify the tracking requirements. Note that the QField panel is only available when the qfieldsync plugin is installed in your QGIS profile.
```
Comment (suricactus) [inline documentation/how-to/tracking.en.md:82]: On line 75 we use single quotes, here we use double quotes. Is there any kind of convention or something?

Comment (suricactus) [reply documentation/how-to/tracking.en.md:82]:
```suggestion
The configuration of a project tracking session happens in the vector layer properties dialog's QField panel. There, you can activate "Tracking Session" and specify the tracking requirements. Note that the QField panel is only available when the QFieldSync plugin is installed in your QGIS profile.
```

File: documentation/how-to/tracking.en.md [hunk 6]
```diff
@@ -69,3 +69,23 @@ on a legend layer linked to an active tracking session and select the
...
+
+!![Activating Autimatic Tracking sessions QFieldSync](../assets/images/automatic-tracking-session.png)
+
+!![Tracking activated automatically in QField](../assets/images/qfield-tracking-session.png,300px)
```
Comment (suricactus) [inline documentation/how-to/tracking.en.md:86]:
```suggestion
!![Activating Automatic Tracking sessions QFieldSync](../assets/images/automatic-tracking-session.png)

!![Tracking activated automatically in QField](../assets/images/qfield-tracking-session.png,300px)
```

One is written in Capitals, the other not, any reason? If it is a name, probably quoting can help?

File: documentation/how-to/tracking.en.md (hunk 5 above)
Comment (SeqLaz) [reply documentation/how-to/tracking.en.md:82]: I think is better with double quote!

File: documentation/how-to/tracking.en.md (hunk 6 above)
Comment (SeqLaz) [reply documentation/how-to/tracking.en.md:86]: Yes, is better to use quotes

**Review by lucienicolier — APPROVED** [review]

**Pull request merged**